src/views/transaction/components/transaction-overview/transaction-overview.view.jsx

```diff
@@ -171,6 +186,21 @@ function TransactionOverview ({
                 />
                 )
           }
+          {
+            transactionType === TxType.Exit && (
+              <div className={classes.exitInfoWrapper}>
+                <div className={classes.exitHelperTextWrapper}>
+                  <InfoIcon className={classes.exitHelperIcon} />
+                  <p className={classes.exitHelperText}>
+                    This step is not revertible. Once initiated, the withdrawal process must be completed.
```

Comment:
revertible  -> reversible

Reply:
Both are valid, I think that I should ask Jenna which one is better!